Due to compat including linux/device.h before we do,
we don't get a chance to #define DEBUG to get our
debugging prints. This patch replaces dev_dbg with
dev_printk to make them show up again.

--- a/drivers/net/wireless/iwlwifi/iwl-debug.c
+++ b/drivers/net/wireless/iwlwifi/iwl-debug.c
@@ -138,8 +138,9 @@ void __iwl_dbg(struct device *dev,
 
 		va_copy(args2, args);
 		vaf.va = &args2;
-		dev_dbg(dev, "%c %s %pV", in_interrupt() ? 'I' : 'U',
-			function, &vaf);
+		dev_printk(KERN_DEBUG, dev, "%c %s %pV",
+			   in_interrupt() ? 'I' : 'U',
+			   function, &vaf);
 		va_end(args2);
 	}
 #endif
